docs/conf.py.in

```diff
@@ -26,6 +26,9 @@ THISDIR='@CMAKE_CURRENT_SOURCE_DIR@'
 sys.path.append(os.path.abspath(os.path.join(THISDIR, 'python_modules')))
 from substitutions import subst as substitutions
 
+# Include custom sphinx extensions
```

Comment:
Please run pep8 on this (ignoring the long lines rule), which is our usual style.

Reply:
There are a few violations against pep8 in the conf.py.in file. Should they be fixed in this pull request?

Reply:
Since we sort of inherit the conf.py format it might be more suitable with another convention. See if it makes sense or not, otherwise I'd be fine with running pep8 on just the module you've written.